### ISSUE TITLE
Add graphics for home page

### ISSUE THREAD

**Issue by MaxGhenis** [PolicyEngine/policyengine-app#26]

**Comment by nikhilwoodruff** [comment]
Added some screenshots a while ago but we could make them fancy device-container-types

**Comment by nikhilwoodruff** [comment]
@robgorman to take this one

**Comment by anth-volk** [comment]
Is this akin to #200?

**Comment by MaxGhenis** [comment]
yeah especially at the top, we need a graphic of the software for the right side like https://slack.com

<img width="1376" alt="image" src="https://github.com/PolicyEngine/policyengine-app/assets/6076111/c93badc4-d9df-4935-a865-3b45b4848314">

then potentially some more below, but that's the most important

**Comment by SEEDART007** [comment]
@MaxGhenis @anth-volk 

![Image](https://github.com/user-attachments/assets/a0253125-cd93-4edd-88c3-989d5c94072f)

![Image](https://github.com/user-attachments/assets/1d2bb1a9-24c0-46b8-afb3-4cecc4c0ee60)
are these good??????

**Comment by anth-volk** [comment]
Those locations are logical, but we're looking to add a feature image akin to the reference Max gave above.

**Comment by MaxGhenis** [comment]
Thank you for filing this issue. Closing as part of the `policyengine-app` (v1) wind-down: policyengine.org is now served by [`policyengine-app-v2`](https://github.com/PolicyEngine/policyengine-app-v2), and this repository is being archived, so this issue is frozen against a codebase that no longer deploys anywhere.

If this is still relevant on the current site, please re-file it against [`policyengine-app-v2`](https://github.com/PolicyEngine/policyengine-app-v2) — we'd welcome it there. Apologies that this sat open through the migration.
